pkgs/misc/emulators/emu2/default.nix

```
@@ -0,0 +1,25 @@
+{ stdenv, lib, fetchFromGitHub }:
+
+stdenv.mkDerivation rec {
+  pname = "emu2";
+  version = "unstable-2020-06-04";
+
+  src = fetchFromGitHub {
+    owner  = "dmsc";
+    repo   = "emu2";
+    rev    = "f9599d347aab07d9281400ec8b214aabd187fbcd";
+    sha256 = "0d8fb3wp477kfi0p4mmr69lxsbgb4gl9pqmm68g9ixzrfch837v4";
+  };
+
+  patchPhase = ''
+    sed -i 's@^PREFIX=.*@PREFIX=$(out)/@g' Makefile
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = with stdenv.lib; {
```

And you can remove `lib` argument attr set.